portfolio.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/styles.css" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,700;1,700&family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,400;1,500;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <title>Homework 2</title>
+</head>
+
+<body>
+  <header>
```

Comment:
add class to `header`

Reply:
Why need we add class to header, if we can use .header? Is it necessary?

Reply:
it is as we use classes as selectors across the project to avoid specificity issues